composer.json

```
@@ -10,7 +10,15 @@
 	"scripts": {
 		"lint": "./vendor/bin/phpcs --standard=phpcs.xml"
 	},
+	"autoload": {
+		"classmap": [
+			"./inc"
+		]
+	},
```

Wondering if we should have a reference to `files` here to demonstrate how to include non-class based php code in the autoload
```suggestion
	    "files" :[]
	},
```

---

sure!